keywords/TestServeriOS.py

```diff
@@ -274,8 +277,11 @@ def start(self, logfile_name):
         # Without --exit, ios-sim blocks
         # With --exit, --log has no effect
         # subprocess.Popen didn't launch the app
+        # output = subprocess.check_output([
+        #    "ios-sim", "--devicetypeid", self.device, "launch", self.app_path, "--exit"
+        # ])
         output = subprocess.check_output([
-            "ios-sim", "--devicetypeid", self.device, "launch", self.app_path, "--exit"
```

Comment:
Changing the order of params would fix the issue ?

Reply:
yes Manasa,